src/applications/ivc-champva/10-7959f-1/config/prefillTransformer.js

```diff
@@ -6,7 +6,10 @@ const prefillTransformer = (pages, formData, metadata) => {
       veteranFullName: formData.veteranFullName,
       veteranAddress: formData.veteranAddress,
       veteranDateOfBirth: formData.veteranDateOfBirth,
-      veteranSocialSecurityNumber: formData.veteranSocialSecurityNumber,
+      veteranSocialSecurityNumber: {
+        ssn: formData.veteranSocialSecurityNumber,
+        vaFileNumber: null,
+      },
```

Comment:
This was the source of the SSN bug - we were setting `veteranSocialSecurityNumber` to a string, so when the form tried to access `veteranSocialSecurityNumber.ssn`, it broke.